PhillipsAuctionHouse/seldon
# L3-1837: add stylelint to project and fix errors

**Jira ticket**

[L3-1837](https://phillipsauctions.atlassian.net/browse/L3-1837)

**Summary**

Stop accidental errors in `scss` files with stylelint

**Change List (describe the changes made to the files)**

- add(package.json): stylelint target
- change(*.scss): fix or ignore any current stylelint errors
- add(stylelint.config.cjs): stylelint config

**Acceptance Test (how to verify the PR)**

- Run `npm run lint`
- Note that no stylelint errors are shown
- Create a rule violation in one of the scss files.  Here's an example of how to fail:
https://stylelint.io/user-guide/rules/font-family-no-missing-generic-family-keyword/
- Make sure your IDE flags the error
- Make sure `npm run lint` catches the error

**Regression Test**

N/A

**Evidence of testing**

```
> stylelint 'src/**/*.scss'


src/components/HeroBanner/_heroBanner.scss
  36:18  ✖  Unexpected missing generic font family  font-family-no-missing-generic-family-keyword

✖ 1 problem (1 error, 0 warnings)
```

<img width="1014" alt="image" src="https://github.com/PhillipsAuctionHouse/seldon/assets/6663002/246ecdd5-a7dc-463b-8dda-6718fe9c01ad">




**Things to look for during review**
